sktime/forecasting/compose/_multiplexer.py

```diff
@@ -81,6 +78,12 @@ class MultiplexForecaster(
     'arima'
     """
 
+    _tags = {
+        "univariate-only": True,
+        "requires-fh-in-fit": True,
```

Comment:
@fkiraly . it  `requires-fh-in-fit` is currently set to `True`. When it is `False` it did not fail. Originally, Multiplexer inherits from `_OptionalForecastingHorizonMixin`.

Reply:
yes, let's move it back, and make note that this may need to be changed (see discussion), in another issue.

Reply:
Done